src/extensions/default/InlineColorEditor/ColorEditor.js

```diff
@@ -215,12 +234,12 @@ define(function (require, exports, module) {
     /** Add event listeners to the format button bar */
     ColorEditor.prototype._bindColorFormatToRadioButton = function (buttonClass, propertyName, value) {
         var handler,
-            _this = this;
+            self = this;
         handler = function (event) {
-            var colorObject, newColor, newFormat;
-            newFormat = $(event.currentTarget).html().toLowerCase().replace("%", "p");
-            newColor = _this.getColor();
-            colorObject = tinycolor(newColor);
+            var newFormat   = $(event.currentTarget).html().toLowerCase().replace("%", "p"),
+                newColor    = self.getColor().toString(),
```

Comment:
@le717 All of the test *currently* pass, but I'm wondering if all calls to  `TinyColor.toString()` should be removed from the unit tests since we know it's not case-sensitive?